src/iotjs_binding.cpp

```diff
@@ -116,13 +119,17 @@ JObject::~JObject() {
 
 
 JObject& JObject::Null() {
-  static JObject null(JVal::Null(), false);
-  return null;
+  if (null == NULL) {
+    null = new JObject(JVal::Null(), false);
```

Comment:
@akosthekiss But lastly, I have just one question to ask you. As far as I know, JS is single thread language, and doens't it mean that this global Null and Undefined can only be accessed one by one??